tools/pages/permit-holders/filter-permit-holders.ts

```diff
@@ -0,0 +1,54 @@
+import { gql } from '@apollo/client'; // gql tag
```

Comment:
1. Could we rename this file to `permit-holders-table.ts`?
2. `FilterPermitHoldersRequest` and `FilterPermitHoldersResponse` should be renamed to `GetPermitHoldersRequest` and `GetPermitHoldersResponse` respectively